### PR TITLE
Add error message for links (SCRD-8302)

### PR DESCRIPTION
**Pull request by eapodaca**
This won't fix the underlying issue but we should be handling errors here.

### CODE REVIEW

**Comment by jeremy-moffitt** [comment]
I got a translation error in the console about `complete.message.linksError` and the error appearing on screen despite not having an error (see attached) 
Screenshot from 2019-03-13 00-16-11

File: src/pages/Complete.js [hunk 1]
```diff
@@ -89,11 +94,14 @@ class Complete extends BaseWizardPage {
           <If condition={this.state.linksLoaing}>
             <h4>{translate('loading.pleasewait')}</h4>
           </If>
-          <If condition={!this.state.linksLoaing}>
+          <If condition={!this.state.linksLoaing && this.state.links}>
             <ul className="body-list">
               {links}
             </ul>
           </If>
+          <If condition={!this.state.linksLoaing && !this.state.link}>
```
Comment (dotNomad) [inline src/pages/Complete.js]: `!this.state.link` should this be `!this.state.links`?

**Comment by eapodaca** [comment]
Updated a few issues.
Not sure what is causing the translation issue the diff shows the `complete.message.linksError` file was added to the bundle. Did you try restarting the dev server?

File: src/localization/bundles/en.json [hunk 2]
```diff
@@ -726,6 +726,8 @@
     "server.replace.compute.failure.delete.cancel.confirm": "Canceling could potentially leave the process of deleting the old compute server {0} in a bad state. It will close the window, and any error messages will no longer be available to view. \n\nProceed with cancellation?",
     "server.replace.compute.failure.disable.cancel.confirm": "Canceling could potentially leave the process of disabling the old compute server {0} in a bad state. It will close the window, and any error messages will no longer be available to view. \n\nProceed with cancellation?",
 
+    "complete.message.linksError": "There was an error fetching the links: {0}",
```
Comment (dotNomad) [inline src/localization/bundles/en.json]: We might want to consider putting a period at the end of this to follow the writing guide.

File: src/pages/Complete.js [hunk 3]
```diff
@@ -42,7 +42,12 @@ class Complete extends BaseWizardPage {
         linksLoaing: false
       });
     } catch(error) {
-      console.log('Unable to retrieve external URLs'); // eslint-disable-line no-console
+      console.log('Unable to retrieve external URLs', error); // eslint-disable-line no-console
+      this.setState({
+        linksError: typeof error === 'string' ? error : error.message | JSON.stringify(error),
```
Comment (guglo) [inline src/pages/Complete.js]: Should be || instead of | otherwise you will get error message as 0

**Pull request merged**